src/deavinaSJun20/TestIntSumClass.java

```diff
@@ -0,0 +1,25 @@
+package deavinaSJun20;
+/*String input = "1Hh9PR34QP";
+Hint : 1+9+3+4 [all digits]
+output : 178*/
+public class TestIntSumClass {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+String name="1Hh9PR34QP";
+
+new TestIntSumClass().intSumClass(name);
+	}
+  void intSumClass(String name) {
```

Comment:
Logic is correct.
Give proper method name so third can understand by reading method name.
rest is good
Keep it up

Reply:
updated file checked in as per review comments . please review

Reply:
Okay merging PR